File: com/pdfjet/PathOperator.java
package com.pdfjet;

/**
 * The operators that paint a path, for example in Page.drawPath.
 */
public enum PathOperator {
    /** Strokes the path. */
    STROKE("S"),
    /** Closes and then strokes the path. */
    CLOSE_AND_STROKE("s"),
    /** Closes and fills the path. */
    FILL("f"),
    /** Closes, fills and then strokes the path. */
    FILL_AND_STROKE("b"),
    /** Like FILL, but uses the even-odd rule. */
    FILL_USING_EVEN_ODD_RULE("f*"),
    /** Like FILL_AND_STROKE, but uses the even-odd rule. */
    FILL_USING_EVEN_ODD_RULE_AND_STROKE("b*");

    /** The operator written to the content stream. */
    final String operator;

    PathOperator(String operator) {
        this.operator = operator;
    }
}

File: Sources/PDFjet/PathOperator.swift
/// The operators that paint a path, for example in Page.drawPath.
public enum PathOperator: String {
    case STROKE = "S"                         // Stroke the path
    case CLOSE_AND_STROKE = "s"                 // Close and then stroke the path
    case FILL = "f"                           // Close and fill the path
    case FILL_AND_STROKE = "b"                  // Close, fill and then stroke the path
    case FILL_USING_EVEN_ODD_RULE = "f*"          // Like 'f' but using even odd rule
    case FILL_USING_EVEN_ODD_RULE_AND_STROKE = "b*" // Like 'b' but using even odd rule
}
